global.d.ts

```diff
@@ -1 +1,2 @@
-declare module "*.pdf";
\ No newline at end of file
+declare module "*.pdf";
```

Comment:
it's weird that you need to declare pdf as a module...there might be problems with the way you import the pdf or with the config of the repository